libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/tracing/TracingFilterConfiguration.kt

```diff
@@ -67,7 +66,6 @@ enum class Target(open val filter: String) {
     MATRIX_SDK_SLIDING_SYNC("matrix_sdk::sliding_sync"),
     MATRIX_SDK_BASE_SLIDING_SYNC("matrix_sdk_base::sliding_sync"),
     MATRIX_SDK_UI_TIMELINE("matrix_sdk_ui::timeline"),
-    MATRIX_SDK_BASE_READ_RECEIPTS("matrix_sdk_base::read_receipts"),
```

Comment:
We could eventually keep this line to be able to manually tweak the tracing configuration from the app.